### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/zh/config/waline.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#643] [Announcements]
# https://plugin-comment2.vuejs.press/zh/config/waline.html

serverURL 类型: string; 必填: 是; Waline 的服务端地址。 emoji 类型: (WalineEmojiInfo WalineEmojiPresets)[] false; 默认值: ['//unpkg.com/@waline/emojis@1.1.0/weibo']; 表情设置，详见 自定义表情 (https://walin...

https://plugin-comment2.vuejs.press/zh/config/waline.html

**Comment by yinuoguan** [comment]
驱蚊器

**Reply by yinuoguan** [reply]
傻了吧你

**Comment by qtdwzAdam** [comment]
111
